### PR TITLE
fix bug for new version of phpstan

### PR DESCRIPTION
**Pull request by kruglikdenis**
Hello, 
In the new version of phpstan, 
for the constructor of the ArrayType class, we need to pass 2 required arguments.

I fixed this problem
and i fixed name of TrueOrFalseBooleanType

### CODE REVIEW

File: src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php [hunk 1]
```diff
@@ -74,16 +74,86 @@ public function __construct()
...
+	private function ap(string $name): ParameterReflection
+	{
+		return $this->createParameterInstance(new ArrayType(new MixedType(), new MixedType(), false), $name, true, false, false);
+	}
+
+	private function fp(string $name): ParameterReflection
+	{
+		return $this->createParameterInstance(new FloatType(), $name, true, false, false);
+	}
+
```
Comment (finwe) [inline src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php]: Why is the position of this block changed?

File: src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php [hunk 2]
```diff
@@ -94,14 +164,14 @@ public function getMethod(ClassReflection $classReflection, string $methodName):
...
-	 * @param bool $public
-	 * @param string $name
-	 * @param mixed[] $parameters
-	 * @param bool $variadic
+	 * @param bool                                $static
+	 * @param bool                                $private
+	 * @param bool                                $public
+	 * @param string                              $name
+	 * @param mixed[]                             $parameters
+	 * @param bool                                $variadic
```
Comment (finwe) [inline src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php]: Please, don't align parameter names and keep used code conventions.

File: src/PHPStan/Faker/Reflection/FakerPropertiesClassReflectionExtension.php [hunk 3]
```diff
@@ -36,7 +37,7 @@ public function __construct()
 			'dateTimeThisDecade' => [new ObjectType(DateTime::class), false, false, true],
 			'dateTimeThisMonth' => [new ObjectType(DateTime::class), false, false, true],
 			'dateTimeThisYear' => [new ObjectType(DateTime::class), false, false, true],
-			'rgbColorAsArray' => [new ArrayType(new IntegerType()), false, false, true],
+			'rgbColorAsArray' => [new ArrayType(new MixedType(), new IntegerType()), false, false, true],
```
Comment (finwe) [inline src/PHPStan/Faker/Reflection/FakerPropertiesClassReflectionExtension.php]: What exactly does this change mean?

File: src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php (hunk 1 above)
Comment (kruglikdenis) [reply src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php]: Sorry, it's mistake. I return back.

File: src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php (hunk 2 above)
Comment (kruglikdenis) [reply src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php]: I fixed it.

File: src/PHPStan/Faker/Reflection/FakerPropertiesClassReflectionExtension.php (hunk 3 above)
Comment (kruglikdenis) [reply src/PHPStan/Faker/Reflection/FakerPropertiesClassReflectionExtension.php]: For ArrayType class we need to pass 2 required parameters. [ArrayType](https://github.com/phpstan/phpstan/blob/master/src/Type/ArrayType.php)

But today i thought and i think that for rgbColorAsArray as type for array key is more suitable IntegerType. I have changed it.

**Pull request merged**